src/status_im/ui/screens/home/styles.cljs

```diff
@@ -15,10 +15,12 @@
   {:flex-shrink 1})
 
 (def last-message-text
-  {:flex        1
-   :align-self  :stretch
-   :line-height 22
-   :color       colors/gray})
+  (merge {:flex        1
+          :align-self  :stretch
+          :line-height 22
+          :color       colors/gray}
+         (when platform/desktop?
```

Comment:
what happens without this?

Reply:
`react-native-desktop` doesn't have an implemented property that restricts `Text` to be one-line, so without this previews of the last message in a chat list are too big if message is long.